### PR TITLE
Page title

### PR DESCRIPTION
**Pull request by betoobernal**
## Summary
- Completed page titles card

## Screenshot
image

## Trello Card
https://trello.com/c/xew31PbR/21-page-titles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" type="text/css" href="scss/application.css">
 </head>
 <body>
-    <h1 class="title">Hello World</h1>
+    <section>
```
Comment (mmamaniwolox) [inline index.html]: This is a good idea, but I dont think you will need this right now

File: index.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
+        <section>
```
Comment (mmamaniwolox) [inline index.html]: you dont need this one

File: index.html [hunk 3]
```diff
@@ -9,6 +9,12 @@
...
+          <header class="header-title">
```
Comment (mmamaniwolox) [inline index.html]: This is not a header, this is just a title element!

File: index.html [hunk 4]
```diff
@@ -9,6 +9,12 @@
...
+            <h1 class="title">Front-End Tools</h1>
```
Comment (mmamaniwolox) [inline index.html]: Lets leave this tag alone and use a pseudo-element in css to add the image and reduce html code

File: scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.header-title {
+  padding: 8rem 10rem 3rem;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: Use `px` instead

File: scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.title {
+  font-size: $text-big;
+  position: relative;
+  width: 100%;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: Im not sure if you will need to use the whole screen width

File: scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  
+  &::before{
+    content: url(../assets/triangle.svg);
+    height: $title-decoration-height;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This constants should be in this file

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/titles.scss (hunk 6 above)
Comment (betoobernal) [reply scss/titles.scss]: I think that I don't need it now, but later sure yes.

File: scss/titles.scss [hunk 8]
```diff
@@ -0,0 +1,18 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  font-size: $text-big;
+  margin: 128px 160px 48px;
```
Comment (mmamaniwolox) [inline scss/titles.scss:6]: This margin is quite wierd, you should do something like `margin: 150px` and no more. At least for now

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**